scalableminds/webknossos
# Allow empty trees

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- create a nml with an empty tree
- upload nml -> tree should be visible
- download annotation -> nml should contain tree

### Issues:
- fixes #3990

------
